test/extensions/filters/http/wasm/wasm_filter_test.cc

```diff
@@ -229,8 +228,7 @@ TEST_P(WasmHttpFilterTest, AccessLog) {
   Buffer::OwnedImpl data("hello");
   EXPECT_EQ(Http::FilterDataStatus::Continue, filter_->decodeData(data, true));
   filter_->onDestroy();
-  StreamInfo::MockStreamInfo log_stream_info;
-  filter_->log(&request_headers, nullptr, nullptr, log_stream_info);
+  filter_->log(&request_headers, nullptr, nullptr, request_stream_info_);
```

Comment:
Why are we using the request_stream_info for the log() call?  That is a different filter.

Reply:
There is one stream info per request / connection. This is an implicit invariant in ABI.